src/validator/maxfee.validator.ts

```diff
@@ -0,0 +1,10 @@
+import {ExpectedError, ValidationContext, Validator} from 'clime';
+
+export class MaxfeeValidator implements Validator<string> {
+    validate(value: string, context: ValidationContext): void {
+        const  tag = /^(([0-9]+\.[0-9]*[1-9][0-9]*)|([0-9]*[1-9][0-9]*\.[0-9]+)|([0-9]*[1-9][0-9]*))$/;
```

Comment:
This is a bit over-complicated for a simple number format, no ?

Reply:
@evias  think the cost should be a non-negative number and a positive integer.